.gitmodules

```diff
@@ -2,17 +2,24 @@
 	path = dg
 	url = https://github.com/mchalupa/dg
 	ignore = untracked
+	branch = master
 [submodule "sbt-slicer"]
 	path = sbt-slicer
 	url = https://github.com/staticafi/sbt-slicer
 	ignore = untracked
+	branch = master
 [submodule "sbt-instrumentation"]
 	path = sbt-instrumentation
 	url = https://github.com/staticafi/sbt-instrumentation
 	ignore = untracked
+	branch = master
 [submodule "klee"]
 	path = klee
 	url = https://github.com/staticafi/klee
+	ignore = untracked
+	branch = 9.0.0-dev
```

Comment:
KLEE won't have the master branch stable?

Reply:
At the moment, KLEE does not contain any branch that's called `master` or `main` and `9.0.0-dev` seems to be the latest development branch at the moment.